fasm2bels/lib/parse_xdc.py

```diff
@@ -7,6 +7,24 @@
                              "net pad line_str line_num params")
 
 
+def to_int_float(s):
+    """ Return the string 's' converted to an int or float if possible, otherwise 
+    just return s """
```

Comment:
Can you add some doctest's for this?  I'm not sure what you are trying to do here.

Reply:
I think both the function name and docstring could be cleaned up.  Maybe "string_to_int_or_float_if_possible"?  That feels verbose, but accurate.  I'm trying to think of a better name.

Reply:
Maybe `to_int_float_or_string`?  Then the docstring summary might be `Convert string to int or float if possibe`, and the detailed explanation could be below, along with some doctest examples.

Reply:
Sounds good.  Since you were already setting the 'DRIVE' property to an integer type, I figured it would be best if the module converted the strings to int/float when possible.  

I'm renaming and adding some doctest examples for it.  I don't see doctest being run as part of the regression testing right now (but I haven't really used it before, so maybe I'm missing it).  Probably good for me to add it.  I don't know best practises with doctest.  I can add a command like this to `make test-py`:

`$(IN_ENV) python -m doctest ./fasm2bels/lib/parse_xdc.py`

Something like this might be better in the long run, but currently this throws a lot of errors:

`$(IN_ENV) find ./fasm2bels -iname "*.py" -exec python -m doctest {} \;`

Or maybe you don't intend for it to be part of the CI?